src/server/ApiRoutes/Hold.js

```diff
@@ -4,6 +4,53 @@ import appConfig from '../../../appConfig.js';
 import User from './User.js';
 import Bib from './Bib.js';
 
+function postHoldAPI(req, cb, errorCb) {
+  // retrieve access token and patron info
+  const accessToken = req.tokenResponse.accessToken;
+  const patronId = req.tokenResponse.decodedPatron.sub;
+  const patronHoldsApi = `${appConfig.api.development}/hold-requests`;
+
+  // get item id and pickup location
+  // NOTE: The implementation for this needs to be redone, or
+  // we may get it directly from the API.
+  let itemId = req.params.itemId;
+  let nyplSource = 'sierra-nypl';
+
+  if (itemId.indexOf('-') >= 0) {
+    const parts = itemId.split('-');
+    itemId = parts[parts.length - 1];
+
+    if (itemId.substring(0, 2) === 'pi') {
+      nyplSource = 'recap-PUL';
+    } else if (itemId.substring(0, 2) === 'ci') {
+      nyplSource = 'recap-CUL';
+    }
+  }
+  itemId = itemId.replace(/\D/g, '');
+  const pickupLocation = req.body.pickupLocation;
+
+  const data = {
+    patron: patronId,
+    recordType: 'i',
+    record: itemId,
+    nyplSource,
+    pickupLocation,
+    // neededBy: "2013-03-20",
+    numberOfCopies: 1,
+  };
+  console.log('Making hold request', data, accessToken);
```

Comment:
Leave it, we'll need it for later when we get the new endpoint to use and when we are manipulating the data.